packages/db/src/schema/user.ts

```diff
@@ -12,6 +12,10 @@ export const user = sqliteTable("user", {
   id: integer("id").primaryKey(),
   tenantId: text("tenant_id", { length: 256 }).unique(), // the clerk User Id
 
+  firstName: text("first_name").default(""),
+  lastName: text("last_name").default(""),
+  email: text("email").default(""),
```

Comment:
It would be nice to store the profile image source url as well so we can use it inside of our application.